config_defaults.yml

```
@@ -557,37 +557,52 @@ Gemfile:
   required:
     ':development':
       - gem: 'json'
+        version: '~> 2.0'
       - gem: 'voxpupuli-puppet-lint-plugins'
-        version: '>= 3.0.0'
+        version: '~> 3.0'
       - gem: 'facterdb'
-        version: '< 2.0.0'
+        version:
+        - '>= 0.8.1'
+        - '< 2.0.0'
```

Version 1.18.0 added Ubuntu 22.04 facts so I'd actually suggest:
```suggestion
        version: '~> 1.18'
```

---

Pinned